frontend/package.json

```diff
@@ -13,10 +13,11 @@
     "enzyme-adapter-react-16": "^1.15.5",
     "jest-sonar-reporter": "^1.3.0",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
+    "react": "^16.14.0",
```

Comment:
did we have to increase the react minimum version for it to work?

Reply:
We were getting a vulnerability warning and other peer dependency warnings. I forget the exact dependency, but one of them required 16.14.0 minimum

Reply:
I'm fine with upgrading the version